atom/common/api/atom_bindings.cc

```diff
@@ -125,6 +126,35 @@ void AtomBindings::Hang() {
     base::PlatformThread::Sleep(base::TimeDelta::FromSeconds(1));
 }
 
+// static
+v8::Local<v8::Value> AtomBindings::GetHeapStatistics(v8::Isolate* isolate) {
+  v8::HeapStatistics v8_heap_stats = {};
+  isolate->GetHeapStatistics(&v8_heap_stats);
+
+  mate::Dictionary dict = mate::Dictionary::CreateEmpty(isolate);
+  dict.Set("totalHeapSize",
```

Comment:
can we make it total_heap_size and similar for all of them, so that anyone using gettHeapStatistics from Node.js v8 can start using this one without breaking.

Reply:
for ref: https://nodejs.org/api/v8.html#v8_v8_getheapstatistics

Reply:
I am against that, we should follow the Electron naming convention